test/benchmarks/Makefile

```diff
@@ -11,7 +11,7 @@ ifndef V
 endif
 
 $(TESTS):
-	$(QUIET_AGGREGATE)python3 ../aggregate.py $(TEST_ARGS) \
+	$(QUIET_AGGREGATE)python3 ../../benchmarks/aggregate.py $(TEST_ARGS) \
```

Comment:
Not quite familiar with the Makefile but I guess it tests by generating a tmp file and compare it with the other checked in files?

Reply:
Correct. See the example run here:  https://github.com/pytorch/xla/pull/6035/commits/bbf01bebf21e
TESTS expands to all the *.csv files in the directory. The arguments passed to aggregate.py are taken from the file name of the CSV file -- this is what `TEST_ARGS` does.